app/components/IncomeListItem.tsx

```diff
@@ -22,15 +25,22 @@ export default function IncomeListItem({ item, index }: ItemProps) {
         router.push(`/ledger/income/edit/${index}`)
     }
 
+    function addItemClicked() {
+        router.push(`/ledger/income/${index}/payment/add`)
+    }
+
     return (
         <Grid row gap className="margin-bottom-5">
-            <Grid col={1}>*</Grid>
-            <Grid col={6} tablet={{col: 9}}>
-                <div>{item.name}</div>
-                <div>{`$${item.amount}`}</div>
+            <Grid col={10}>
                 <div>{item.description}</div>
+                <div>{item.business}</div>
+                <div>
+                    <ul>
+                        {payments}
+                    </ul>
+                </div>
             </Grid>
-            <Grid col={5} tablet={{col: 2}}>
+            <Grid col={2}>
                 <ModalToggleButton modalRef={ref} opener unstyled className="margin-bottom-2">{t('list_income_delete_button')}</ModalToggleButton>
```

Comment:
Not from this PR but there should probably be some space between these buttons. 
<img width="917" alt="Screenshot 2024-09-20 at 1 20 14 PM" src="https://github.com/user-attachments/assets/48fb6ae9-dc45-4d9c-9aa1-0b4404ef9521">